extension.js

```diff
@@ -169,22 +171,22 @@ const GTileStatusButton = new Lang.Class({
     _init: function(classname) {
         this.parent(0.0, "gTile", false);
 
-        this.actor.add_style_class_name(classname);
+        this.add_style_class_name(classname);
```

Comment:
This fails for me. See https://github.com/gTile/gTile/pull/110#issuecomment-586858487

Reply:
Yes... this is bad problem. On gnome-shell 3.32 this line generate exception with warning about deprecating actor in ST button.
I fixed by removing agent, but had no way to test on earlier versions like 3.28
Obviously, you have a correct fix.
Most likely code needs to discriminate for version of gnome-shell, use actor for version below 3.32 and no actor above.
Anu idea how to get gnome-shell version?

Reply:
`imports.misc.config.PACKAGE_VERSION`

Reply:
Thanks!
Will post fix. I cannot commit from office, can do only over weekend.

Reply:
Yes, you can keep actor in this.actor.connect, it is not causing deprecation message.
No need to check for gnome-shell version, it is working on most recent. And, apparently it is also working on previous versions.